### ISSUE TITLE
rsslSubmit java.lang.NullPointerException

### ISSUE THREAD

**Issue by tof247** [Refinitiv/Real-Time-SDK#281]
Hello EMA team,  

With EMA java 3.7.3.0 regularly we have below call stack:

```
java.lang.NullPointerException: null
    at com.refinitiv.ema.access.SingleItem.rsslSubmit(ItemCallbackClient.java:3074)
    at com.refinitiv.ema.access.SingleItem.open(ItemCallbackClient.java:2936)
    at com.refinitiv.ema.access.ItemCallbackClient.registerClient(ItemCallbackClient.java:2210)
    at com.refinitiv.ema.access.OmmBaseImpl.registerClient(OmmBaseImpl.java:561)
    at com.refinitiv.ema.access.OmmConsumerImpl.registerClient(OmmConsumerImpl.java:255)
```

Can you please help us to identify the possible root case of this issue.

Thanks and regards

**Comment by vlevendel** [comment]
@tof247 Please add handling in your consuming application for receiving a Login/Closed/Suspect.  https://github.com/Refinitiv/Real-Time-SDK/blob/master/Java/Ema/Examples/src/main/java/com/refinitiv/ema/examples/training/consumer/series300/ex333_Login_Streaming_DomainRep/Consumer.java has some helpful hints.  

A fix for this issue will be made in an upcoming release.

**Comment by vlevendel** [comment]
@tof247 This is addressed in RTSDK 2.2.2.L1.
